src/Components/Filtering/Filtering.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable max-len */
```

Comment:
If you can, delete this disable and instead make really long lines in the code multiline where necessary.